Extensions/BBText/bbtextruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * The PIXI.js renderer for the BBCode Text runtime object.
+ *
+ * @class BBTextRuntimeObjectPixiRenderer
+ * @constructor
+ * @param {gdjs.BBTextRuntimeObject} runtimeObject The object to render
+ * @param {gdjs.RuntimeScene} runtimeScene The gdjs.RuntimeScene in which the object is
+ */
+gdjs.BBTextRuntimeObjectPixiRenderer = function(runtimeObject, runtimeScene) {
+  this._object = runtimeObject;
+
+  // Load (or reset) the text
+  if (this._pixiObject === undefined) {
+    this._pixiObject = new MultiStyleText(runtimeObject._text, {
+      default: {
+        fontFamily: runtimeObject._fontFamily,
+        fontSize: runtimeObject._fontSize + 'px',
+        fill: runtimeObject._color,
+        tagStyle: 'bbcode',
+        wordWrap: runtimeObject._wordWrap,
+        wordWrapWidth: runtimeObject._wrappingWidth,
+        align: runtimeObject._align,
+      },
+    });
+
+    this._object.hidden = !runtimeObject._visible;
```

Comment:
I'm going to merge this, but can you take a look at this line? This is modifying the object, which should not happen in a XxxPixiRenderer class (renderers are here to render, not to modify the object :)).
It's also weird because `this._object` *is* `runtimeObject` => all of this can and should be done in the runtime object directly, which is the "source of truth", containing the state of the object.